### ISSUE TITLE
Where is the location of Finetuning Loss code?

### ISSUE THREAD

**Issue by prstrive** [TWJianNuo/EdgeDepth-Release#10]
Thanks for your perfect work! But in the paper, you finetune your model with 3 more epochs. But I don't find the fine-tuning progress in your code! Could you please explain to me?
